python_src/src/lamp_py/tableau/jobs/gtfs_rail.py

```diff
@@ -40,10 +40,13 @@ def create_parquet(self, db_manager: DatabaseManager) -> None:
         if os.path.exists(self.local_parquet_path):
             os.remove(self.local_parquet_path)
 
+        db_batch_size = 1024 * 1024 / 2
+
         db_manager.write_to_parquet(
             select_query=sa.text(self.create_query),
             write_path=self.local_parquet_path,
             schema=self.parquet_schema,
+            batch_size=db_batch_size,
```

Comment:
Based on ECS Health dashboard, memory utilization appeared to peak somewhere between 70-90% on our ECS with 16GB of memory. A reduced `batch_size` should cut that in half.